lib/config/config_test.go

```diff
@@ -391,24 +398,28 @@ reporting:
 	assert.Equal(t, password, config.Kafka.Password)
 	assert.True(t, config.SharedTransferConfig.TypingSettings.CreateAllColumnsIfAvailable)
 
-	var foundOrder bool
-	var foundCustomer bool
+	orderIdx := -1
+	customerIdx := -1
+	for idx, topicConfig := range config.Kafka.TopicConfigs {
+		topicConfig.Load()
 
-	for _, topicConfig := range config.Kafka.TopicConfigs {
 		assert.Equal(t, topicConfig.Database, "customer")
 		assert.Equal(t, topicConfig.Schema, "public")
 
 		if topicConfig.TableName == "orders" {
-			foundOrder = true
+			orderIdx = idx
 		}
 
 		if topicConfig.TableName == "customer" {
-			foundCustomer = true
+			customerIdx = idx
 		}
 	}
 
-	assert.True(t, foundCustomer)
-	assert.True(t, foundOrder)
+	assert.True(t, config.Kafka.TopicConfigs[orderIdx].ShouldSkip("d"))
+	assert.True(t, config.Kafka.TopicConfigs[customerIdx].ShouldSkip("c"))
+
+	assert.True(t, customerIdx >= 0)
```

Comment:
I feel like this check should happen before `assert.True(t, config.Kafka.TopicConfigs[customerIdx].ShouldSkip("c"))`

Reply:
Shure

Reply:
It'll all get evaluated, we're not using require which would fatal on the first failed assertion

Reply:
Ohh that's right, carry on!